functional-tests/features/testing-api/testing_post_form_data.feature

```diff
@@ -3,6 +3,14 @@ Feature: Test server post request using form data
     I want to test restApi post request
 
     Scenario: REST get example test
-        Given I send "POST" request on "postFormDataEndpoint" endpoint using form data:
-        | name | adam1 |
+        Given I set request headers:
+            | Content-Type | multipart/form-data |
+        And I send "POST" request on "postFormDataEndpoint" endpoint using form data:
+            | name | adam1 |
+        Then the response code should be "201"
+
+    Scenario: REST upload example file
+        Given I send "POST" request on "upload" endpoint using form data:
+            | name   | test |                         |
+            | myFile |      | ./data/kakunin_logo.png |
```

Comment:
Use path.resolve instead of relative path

Reply:
I use there `fs.createReadStream` and it works with `data/kakunin_logo.png`